ddtrace/propagation/http.py

```diff
@@ -27,7 +30,23 @@
 POSSIBLE_HTTP_HEADER_ORIGIN = frozenset([HTTP_HEADER_ORIGIN, get_wsgi_header(HTTP_HEADER_ORIGIN).lower()])
 
 
-class HTTPPropagator(object):
+class Propagator(six.with_metaclass(abc.ABCMeta)):
```

Comment:
This is already defined in `ddtrace.opentracer.propagation.propagator`. Would it make sense to unify the definitions?